examples/vmi-with-virt-xml-sidecar-hook.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    hooks.kubevirt.io/hookSidecars: '[{"args": ["--version", "v1alpha2", "--args",
+      "--disk=iotune.total_bytes_sec=52428800,iotune.total_iops_sec=1000|--edit=all"],
```

Comment:
While passing `--disk` first and `--edit` second will work, it's IMO cleaner to have the operation first. The virt-xml manual page also documents the usage as such.